src/clincoded/static/components/variant_central/interpretation/segregation/masterTable.js

```diff
@@ -0,0 +1,356 @@
+'use strict';
+
+// stdlib
+import React from 'react';
+import PropTypes from 'prop-types';
+import createReactClass from 'create-react-class';
+
+// Internal
+import { masterTable, extraEvidence } from './segregationData';
+import { external_url_map } from '../../../globals';
+import { EvidenceModalManager } from './evidenceModalManager';
+import { DeleteEvidenceModal} from './deleteEvidenceModal';
+import { getAffiliationName } from '../../../../libs/get_affiliation_name';
+
+let MasterEvidenceTable = createReactClass({
+    propTypes: {
+        evidence_arr: PropTypes.array,              // All pieces of evidence added to this variant
+        affiliation: PropTypes.object,              // User's affiliation
+        session: PropTypes.object,                  // Session object
+        viewOnly: PropTypes.bool,                   // If the page is in read-only mode
+        deleteEvidenceFunc: PropTypes.func,         // Function to call to delete an evidence
+        evidenceCollectionDone: PropTypes.func,     // Fucntion to call to add or edit an existing one
+        canCurrUserModifyEvidence: PropTypes.func   // Funcition to check if current logged in user can modify the given evidence
+    },
+
+    getInitialState() {
+        return {
+            rows: this.props.evidence_objects
```

Comment:
```suggestion
            rows: this.props.evidence_arr
```
Reason: **evidence_objects** doesn't appear to be passed to **MasterEvidenceTable** and isn't declared as a prop